CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Node navbar: to accompany all node pages
```

Comment:
This should be called Project navbar. A node doesn't make sense intuitively as a backend object. It's called a project on the front-end, so we should stick with that.